src/components/ColorPicker/ColorPicker.vue

```diff
@@ -32,10 +32,55 @@ actual pickers:
 
 ### Usage
 
+* Using v-model and passing in an HTML element that will be treated as a trigger:
+
+```vue
+<template>
+	<ColorPicker v-model="color">
+		<button> Click Me </button>
+	</ColorPicker>
```

Comment:
Should we have a tiny span here, that just contains `{{ color }}`? 
To show when it updates.

Reply:
@georgehrke there might be a problem with that. That :color, is only for the 'advanced' color picker, the 'simple' color picker just emits an event with the color as a payload. So basically the color value in the data is updated only when you change it with the advanced one.

Reply:
..which was actually a bug, It's fixed now :)